LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -1217,22 +1217,22 @@ extension DoseStore {
                 return
             }
 
-            do {
-                let doses = try self.getNormalizedReservoirDoseEntries(start: startDate)
-                let result = InsulinValue(
-                    startDate: doses.first?.startDate ?? self.currentDate(),
-                    value: doses.totalDelivery
-                )
+            self.getNormalizedDoseEntries(start: startDate) { (result) in
+                switch result {
+                case .success(let doses):
+                    let result = InsulinValue(
+                        startDate: doses.first?.startDate ?? self.currentDate(),
```

Comment:
`startDate` will be missing if there are no doses retrieved, in which case total delivery will be 0; the date is used to show a 'since' field on the InsulinDeliveryTableViewController.